random-beans/src/main/java/io/github/benas/randombeans/util/ReflectionUtils.java

```diff
@@ -165,6 +166,24 @@ public static boolean isParameterizedType(final Type type) {
     }
 
     /**
+     * Check if a type is an unbounded wildcard type
+     * @param parameterizedType the ParameterizedType to check
+     * @return true if the ParameterizedType is an unbounded wild card type, false otherwise
+     */
+    public static boolean isUnboundedWildcardType(final ParameterizedType parameterizedType) {
```

Comment:
Hey, this is better than the current static method `isWildcardType` of the ReflectionUtils class. But I think that testing if the type is instance of WildcardType is fair enough. Can you please try to use it in replacement of `isUnboundedWildcardType` and see if your tests pass? If so, it would be great to update the PR to keep only tests, they are more complete than the [poor](https://github.com/benas/random-beans/blob/master/random-beans/src/test/java/io/github/benas/randombeans/CollectionPopulationTest.java#L145) [ones](https://github.com/benas/random-beans/blob/master/random-beans/src/test/java/io/github/benas/randombeans/MapPopulationTest.java#L152) I've created .. Don't hesitate to get rid of them and their corresponding beans